packaging/docker/run.sh

```diff
@@ -80,9 +80,9 @@ if [ "${EUID}" -eq 0 ]; then
 
   if [ -n "${PGID}" ]; then
     echo "Creating docker group ${PGID}"
-    addgroup --gid "${PGID}" "docker" || echo >&2 "Could not add group docker with ID ${PGID}, its already there probably"
+    addgroup --gid "${PGID}" "docker" || echo >&2 "Could not add group docker with ID ${PGID}, probably one already exists"
     echo "Assign netdata user to docker group ${PGID}"
-    usermod --append --groups "docker" "${DOCKER_USR}" || echo >&2 "Could not add netdata user to group docker with ID ${PGID}"
+    usermod --append --groups "$(getent group "${PGID}" | cut -d: -f1)" "${DOCKER_USR}" || echo >&2 "Could not add netdata user to group docker with ID ${PGID}"
```

Comment:
```suggestion
    GROUP_NAME=$(getent group "${PGID}" | cut -d: -f1)
    echo "Assign netdata user to group ${GROUP_NAME} (GID: ${PGID})"
    usermod --append --groups "${GROUP_NAME}" "${DOCKER_USR}" || echo >&2 "Could not add netdata user to group ${GROUP_NAME} with ID ${PGID}"
```

The error message in the usermod command still references "docker group" even though you might be adding to a different group.

Reply:
@ilyam8 I addressed it, but also noticed some inconsistency with another part of the code which I fixed too. Hope you like it.